client/src/components/helpers/toggle-button.tsx

```diff
@@ -36,7 +36,7 @@ export default function ToggleButton({
     width: '20px'
   };
   return (
-    <Fragment>
+    <>
```

Comment:
The component content is wrapped inside a `BSBG`, so a fragment here is not needed.

Reply:
Done